src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -399,15 +401,15 @@ public void zadd(final byte[] key, final double score, final byte[] member) {
     sendCommand(ZADD, key, toByteArray(score), member);
   }
 
-  public void zaddBinary(final byte[] key, final Map<byte[], Double> scoreMembers) {
+  public void zadd(final byte[] key, final double score, final byte[] member,
+      final ZAddParams params) {
+    sendCommand(ZADD, params.getByteParams(key, toByteArray(score), member));
+  }
 
+  public void zaddBinary(final byte[] key, final Map<byte[], Double> scoreMembers) {
```

Comment:
I know it has nothing to do with this PR. But I was looking at the `zaddBinary` method and I was thinking: "why is it called zaddBinary if zadd is already binary?". Then I realized that the difference is that `zaddBinary` accepts multiple score members. 

Maybe we can take the opportunity to rename this method to just `zadd`?. In Client.java the method that calls `zaddBinary` is called `zadd`.